kernel/src/main/scala/cats/kernel/std/option.scala

```diff
@@ -0,0 +1,68 @@
+package cats.kernel
+package std
+
+package object option extends OptionInstances
+
+trait OptionInstances extends OptionInstances1 {
+  implicit def optionOrder[A: Order] = new OptionOrder[A]
+  implicit def optionMonoid[A: Semigroup] = new OptionMonoid[A]
```

Comment:
there are at least two other Monoids: `left.orElse(right)` `right.orElse(left)`. Should we expose those (maybe not as the default?)